NitroxClient/MonoBehaviours/Gui/Chat/PlayerChatEntry.cs

```diff
@@ -42,7 +43,7 @@ public void OnGUI()
 
                 if (Event.current.isKey && Event.current.keyCode == KeyCode.Escape)
                 {
-                    Hide();
+                    Hide(true);
```

Comment:
I'm not too keen on optional flags for methods like this.  I think it would be cleaner to do

```
chatManager.HideLog();
Hide();
```

instead of passing in a parameter.  Mainly because this is the only place that passes in the parameter.